src/ontology/config/prefixes.csv

```diff
@@ -243,14 +242,12 @@ HGNC,http://identifiers.org/hgnc/
 MEDDRA,http://identifiers.org/meddra/
 MEDGEN,http://identifiers.org/medgen/
 SCTID,http://identifiers.org/snomedct/
-UMLS,http://linkedlifedata.com/resource/umls/id/
 orcid,https://orcid.org/
 swrl,http://www.w3.org/2003/11/swrl#
 semapv,https://w3id.org/semapv/vocab/
 HGNC_SYMBOL,https://www.genenames.org/data/gene-symbol-report/#!/hgnc_id/
 HGNC,https://identifiers.org/hgnc/
 ncbi.gene,https://www.ncbi.nlm.nih.gov/gene/
-OMIMPS,https://omim.org/phenotypicSeries/PS
```

Comment:
Not a duplicate (different IRI, see comment in issue)

Reply:
This one is a dupe!:
https://github.com/monarch-initiative/mondo-ingest/blob/adc201572f76b24fa80ff907ee726932249d9b87/src/ontology/config/prefixes.csv#L234
https://github.com/monarch-initiative/mondo-ingest/blob/adc201572f76b24fa80ff907ee726932249d9b87/src/ontology/config/prefixes.csv#L253

Reply:
I looked at [your comment](https://github.com/monarch-initiative/mondo-ingest/pull/503#pullrequestreview-2028050557), and I think it is invalidated now.

@matentzn Can you review / confirm that these are indeed pure duplicate rows?

Let me know, even that being the case, if you'd like me to run a build. And, if so, then I wonder if it might make sense to combine benign changes like these into a single PR so I don't have to run as many builds.

Reply:
Yep, the problem was that github preview uses lazy load; i just opened the file, hit browser search, didnt find the string and said "no dupe". If I would have scrolled, I would have seen it. Sorry!